Frontend/src/views/PointDetails.vue

```diff
@@ -0,0 +1,113 @@
+<template>
+
+    <div id = "pointdisplay" class="component" >
+        <h1>Point Details</h1>
+        <br />
+        <v-form>
+            <tt>Created At:</tt>
+            <strong>
+                {{createdAt}}
+            </strong>
+            <br /><br />
+            <tt>Point Name:</tt>
+    
+            <strong>
+                {{name}}
+            </strong>
+
+            <br /><br />
+            <tt>Point Description:</tt>
+            <input type ="text"  v-bind:placeholder= "description" >
+
+            <br />
+            <br />
+            <tt>Longitude:</tt>
+            <strong>
+                {{long}}
+            </strong>
+            <br />
+            <br />
+            <tt>Latitude:</tt>
+            <strong>
+                {{lat}}
+            </strong>
+            <br />
+            <br />
+            <tt>Last Modified At:</tt>
+            <strong>
+                {{lastModifiedAt}}
+            </strong>
+            <br />
+            <br />
+            <tt>Point Id:</tt>
+            <strong>
+                {{id}}
+            </strong>
+            <br />
+           
+        </v-form>
+
+    </div>
+
+
+</template>
+
+<script>
+
+import {getPoint} from '../services/pointServices'
+import { checkSession } from '../services/authorizationService';
+
+export default {
+  name: 'PointDetails',
+  data: function() {
+    return {
+      id: null,
+      url: null,
+      createdAt: null,
+      lastModifiedAt: null,
+      name: null,
+      description: null,
+      long: null,
+      lat: null
+
+    }
+  }, 
+  methods: {
+     getPointDetails: function()
+     { 
+         this.url = window.location.href
+         this.id = this.url.substring(this.url.length-36,this.url.length)
```

Comment:
there is an easy way to get a header value of a route parameter by using
this.$route.query.pointId

Reply:
i will change this